src/js/components/dashboard/impacts/ImpactGauge.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * ImpactGauge.jsx
+ * Created by Daniel Boos 4/1/2020
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    submissionData: PropTypes.object,
+    openModal: PropTypes.func.isRequired,
+    level: PropTypes.string
+};
+
+const defaultProps = {
+    submissionData: {},
+    level: 'high'
+};
+
+export default class ImpactGauge extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.onClick = this.onClick.bind(this);
+    }
+
+    onClick() {
+        if (this.props.submissionData.rules) {
+            this.props.openModal(this.props.submissionData.rules, this.props.level);
+        }
+    }
+
+    render() {
+        return (
+            <div
+                className="impact-section"
+                onClick={this.onClick}
+                onKeyDown={this.onClick}
```

Comment:
We'll want to use the [createOnKeyDownHandler helper](https://github.com/fedspendingtransparency/data-act-broker-web-app/blob/dev/src/js/helpers/util.js#L157) to only trigger the action for enter and spacebar instead of any key.